DiscordBot/Modules/UserModule.cs

```diff
@@ -459,6 +459,42 @@ public async Task TopKarma()
             await ReplyAsync(embed: embed).DeleteAfterTime(minutes: 1);
         }
 
+        [Command("TopKarmaWeekly"), Priority(5)]
```

Comment:
These weekly/monthly/yearly commands we should probably exclude from !help as it is already quite long. (or we can keep them, another 3 won't kill anyone) 

the [Priority(n)] attribute is hijacked for ordering in the !help command. But we can replace that with [HideFromHelp] which will mean it won't show up.

We could edit the !TopKarma summary to include in description that the weekly/yearly exist, or let people find it on their own.